tests/plots/test_bar.py

```
@@ -0,0 +1,321 @@
+@pytest.fixture()
+def sample_dataframe():
+    """A sample dataframe for testing."""
+    data = {
+        "product": ["A", "B", "C", "D"],
+        "sales_q1": [1000, 1500, 2000, 2500],
+        "sales_q2": [1100, 1600, 2100, 2600],
+    }
+    return pd.DataFrame(data)
+
+
+@pytest.fixture()
```

Ruff

Use `@pytest.fixture` over `@pytest.fixture()`

Remove parentheses

(PT001)